src/db.rs

```diff
@@ -49,6 +52,7 @@ impl DbInner {
             table_store,
             memtable_flush_notifier,
             db_stats: Arc::new(DbStats::new()),
+            sst_format,
```

Comment:
I think it would be cleaner to add an `estimate_size` method to `TableStore` than to pass the format to `DbInner`.